src/platforms/javascript/guides/nextjs/manual-setup.mdx

```diff
@@ -52,9 +52,9 @@ export default withSentry(handler);
 
 You can include your DSN directly in these two files, or provide it in either of two environment variables, `SENTRY_DSN` or `NEXT_PUBLIC_SENTRY_DSN`.
 
-## Extend Default Webpack Usage for Source Maps
+## Extend Default Webpack Usage
 
-Extend the default Next.js usage of Webpack to generate source maps and upload them to Sentry. Include the following in `next.config.js`:
+Extend the default Next.js usage of Webpack to initialize the SDK, capture errors, and generate and upload source maps to Sentry. Include the following in your `next.config.js`:
```

Comment:
I feel like to make this really clear, it's not going to work to cram it all into once sentence. What do you think of the following?

```suggestion
Use `withSentryConfig` to extend the default Next.js usage of Webpack. This will do two things:
- Generate and upload source maps to Sentry, so that your stacktraces contain original, demangled code.
- Automatically call the code in `sentry.server.config.js` and `sentry.client.config.js`, at server start up and client page load, respectively. Using `withSentryConfig` is the only way to guarantee that the SDK is initialized early enough to catch all errors.

Include the following in your `next.config.js`:
```

Reply:
I like it. Changed the bullet order to have them sorted by relevance.

Reply:
👍🏻